nubis/puppet/files/interface-fixup

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+[ -e /usr/local/lib/nubis/nubis-lib.sh ] && . /usr/local/lib/nubis/nubis-lib.sh || exit 1
+
+mac_eth0=$(__get_mac eth0)
+ip_addr=$(curl -s -fq "http://169.254.169.254/latest/meta-data/network/interfaces/macs/${mac_eth0}/local-ipv4s")
+
+if [[ -f '/etc/consul/interface.json' ]]; then
```

Comment:
This works only once, then you lose your replace marker. Instead, especially with a very simple file, I much prefer the pattern of just

cat <<EOF> /etc/consul/interface.json
{
  "advertise_addr": "$ip_addr"
}
EOF